pkg/schedule/windowed_schedule_test.go

```diff
@@ -100,14 +147,23 @@ func TestWindowedSchedule(t *testing.T) {
 					time.Sleep(w.Interval * 2)
 				}
 			}
-			// we should have either 16 or 17 minus 3 missed
-			So(len(r), ShouldBeBetweenOrEqual, 10, 12)
+			// we should have either 12 or 14 results minus 4 missed
```

Comment:
windowed width has been changed to 200ms to be more corresponding to the other tests-cases.